analytics_data_api/v0/tests/test_connections.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+from django.test.utils import override_settings
+from elasticsearch.exceptions import ElasticsearchException
+from mock import patch
+
+from analytics_data_api.v0.connections import BotoHttpConnection, ESConnection
+
+
+class ESConnectionTests(TestCase):
+
+    @override_settings(ELASTICSEARCH_CONNECTION_DEFAULT_REGION='region_123')
+    def test_region(self):
+        connection = ESConnection('mockservice.cc-zone-1.amazonaws.com',
+                                  aws_access_key_id='access_key',
+                                  aws_secret_access_key='secret')
+        self.assertEqual(connection.auth_region_name, 'region_123')
+
+
+@override_settings(ELASTICSEARCH_AWS_ACCESS_KEY_ID='access_key')
+@override_settings(ELASTICSEARCH_AWS_SECRET_ACCESS_KEY='secret')
+class BotoHttpConnectionTests(TestCase):
+
+    @patch('analytics_data_api.v0.connections.ESConnection.make_request')
+    def test_perform_request_success(self, mock_response):
+        mock_response.return_value.status = 200
+        connection = BotoHttpConnection()
+        status, _header, _data = connection.perform_request('get', 'http://example.com')
+        self.assertEqual(status, 200)
```

Comment:
I'm feeling a bit torn about writing tests for the logging.  `elasticsearch-py` [tests that functionality](https://github.com/elastic/elasticsearch-py/blob/master/test_elasticsearch/test_connection.py#L148) and testing it on our end end feels redundant and doesn't place trust in the library.  At the same time, the logging is a side effect of what it is that we're calling explicitly in our method, and thus arguably should be part of our tests.  I'm leaning to leaving it untested, but am interested in your thoughts.